op-challenger/game/fault/trace/alphabet/provider.go

```diff
@@ -15,12 +15,12 @@ import (
 
 var (
 	ErrIndexTooLarge = errors.New("index is larger than the maximum index")
-	absolutePrestate = common.Hex2Bytes("0000000000000000000000000000000000000000000000000000000000000060")
 )
 
 // AlphabetTraceProvider is a [TraceProvider] that provides claims for specific
 // indices in the given trace.
 type AlphabetTraceProvider struct {
+	AlphabetPrestateProvider
```

Comment:
Small nit but it feels weird to have a `NewPrestateProvider` function for `AlphabetPrestateProvider` but then just use the null/default instance here.  It works because it is completely stateless, but I wonder if we remove the constructor and add a note on the `AlphabetPrestateProvider` struct definition that it's intended to be stateless.

Reply:
Agreed, I did this, adding a comment about the statelessness of the `AlphabetPrestateProvider` and its use of a hardcoded pre-state hash.